### PR TITLE
cmd: Add --include-files option

### PR DESCRIPTION
**Pull request by xantares**
also remove the --ext-list-file option which was ignored

### CODE REVIEW

**Comment by xantares** [comment]
ping @anthony-tuininga @marcelotduarte

**Comment by marcelotduarte** [comment]
@xantares, I believe that @anthony-tuininga will soon accept your patch.

File: cx_Freeze/main.py [hunk 1]
```diff
@@ -94,11 +94,10 @@ def ParseCommandLine():
             dest = "excludeModules",
             metavar = "NAMES",
             help = "comma separated list of modules to exclude")
-    parser.add_option("--ext-list-file",
-            dest = "extListFile",
-            metavar = "NAME",
-            help = "name of file in which to place the list of dependent files "
-                   "which were copied into the target directory")
```
Comment (anthony-tuininga) [inline cx_Freeze/main.py:101]: Why are you removing the --ext-list-file option?

Comment (xantares) [reply cx_Freeze/main.py:101]: @anthony-tuininga  this option is unused: extListFile is not passed to Freezer, and I see no option in Freezer that could match its description, so I felt its best to remove it than silently ignore it

Comment (xantares) [reply cx_Freeze/main.py:101]: @anthony-tuininga are we good ?

Comment (anthony-tuininga) [reply cx_Freeze/main.py:101]: Yes. I see that indeed the option is unused.

**Pull request merged**